src/unix/sunos.c

```diff
@@ -216,8 +228,9 @@ void uv__io_poll(uv_loop_t* loop, int timeout) {
     SAVE_ERRNO(uv__update_time(loop));
 
     if (events[0].portev_source == 0) {
-      if (timeout == 0)
+      if (timeout == 0) {
```

Comment:
Yeah, sorry. I thought I got all of those but I obviously missed this one. 

I will remark that it was somewhat annoying to have to add and remove braces every time I needed to add/remove debugging to a single-line body. (Emacs can't magically handle that for me, can it?)